packages/features/insights/components/RoutingFormResponsesTable.tsx

```diff
@@ -41,6 +63,17 @@ type RoutingFormTableRow = {
   [key: string]: any;
 };
 
+type FieldCellValue = { label: string; value: string };
+
+const ZResponseValues = z.array(
+  z.object({
+    label: z.string(),
+    value: z.string(),
+  })
+);
+
+type ResponseValues = z.infer<typeof ZResponseValues>;
```

Comment:
If not needed, don't rely on z.infer.

```suggestion
type ResponseValues = FieldCellValue[];
```